dusty/cli/test.py

```diff
@@ -21,16 +22,23 @@
 from docopt import docopt
 
 from ..payload import Payload
-from ..commands.test import (run_app_or_lib_tests, test_info_for_app_or_lib, pull_repos_and_sync_commands,
-                             ensure_valid_suite_name)
+from ..commands.test import (run_app_or_lib_tests, test_info_for_app_or_lib, pull_repos_and_sync,
+                             ensure_valid_suite_name, run_all_app_or_lib_suites)
 
 def main(argv):
     args = docopt(__doc__, argv, options_first=True)
-    if not args['<suite_name>']:
-        return Payload(test_info_for_app_or_lib, args['<app_or_lib_name>'])
-    else:
+    if args['<suite_name>'] == '--all':
```

Comment:
This is crazy hacky, if something has `--` in front of it it should definitely be an option, not an arg. If we're going to do it this way I think it'd be better to just make `all` a reserved suite name.